src/main/java/net/neoforged/neoforge/common/extensions/ILevelReaderExtension.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common.extensions;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Registry;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.world.level.LevelReader;
+
+public interface ILevelReaderExtension extends RegistryAccess {
+    private LevelReader self() {
+        return (LevelReader) this;
+    }
+
+    default boolean isAreaLoaded(BlockPos center, int range) {
+        return self().hasChunksAt(center.offset(-range, -range, -range), center.offset(range, range, range));
+    }
+
+    /**
+     * Delegates the implementation of RegistryAccess to enable modders to access HolderLookup.Provider methods
+     * directly without traversing through level.registryAccess().
+     * For additional utilities, see {@link IHolderLookupProviderExtension}
+     */
+
```

Comment:
Why is there an empty line here?

Reply:
Mainly to keep it away from the javadoc directly on the method, as its more of a note for maintainers instead of modders.

Reply:
It's still part of the method javadoc if it's a javadoc comment, even with additional empty lines.